skrub/_dataframe/_common.py

```diff
@@ -825,6 +826,27 @@ def _to_categorical_polars(col):
     return _cast_polars(col, pl.Categorical())
 
 
+@dispatch
+def is_all_null(col):
+    raise NotImplementedError()
+
+
+@is_all_null.specialize("pandas", argument_type="Column")
+def _is_all_null_pandas(col):
+    return bool(col.isna().all())
+
+
+@is_all_null.specialize("polars", argument_type="Column")
+def _is_all_null_polars(col):
+    if col.dtype == pl.Null:
+        return True
+    elif col.dtype.is_numeric() and col.is_nan().all():
```

Comment:
there could be a mix of nan and null which I think would fall through the cracks here.

For simplicity I think we could do:

```python
def is_all_null(col): return all(is_null(col))
```

but if you really want the specialization for polars to get the speedup with null count it could be

```python
def _is_all_null_polars(col):
    if col.dtype == pl.Null:
        return True
    if col.null_count() == col.len():
        return True
    return all(is_null(col))
```